rich/console.py

```diff
@@ -673,18 +673,6 @@ def __init__(
             if legacy_windows is None
             else legacy_windows
         )
-        if width is None:
```

Comment:
This block is not redundant. It the size from being calculated ever time the size property is used.

Reply:
Thanks for review! I check it again, it definitely will result in different values in some cases. Actually, I just want to clean up [Line 662-664](https://github.com/Textualize/rich/blob/1772458f80924c78fa3a8d0260bca9762ad86e18/rich/console.py#L662-L664) at first, then I glanced some codes in `def size` are very similar, that's my fault.

So how about current updates?